cinch/roles/jenkins_master/tasks/iptables.yml

```diff
@@ -48,13 +80,19 @@
     state: present
   register: output_port_forward
 
+- name: set iptables to reject all other connections
```

Comment:
Is it possible to set this as the default policy for the INPUT chain rather than as an explicit ordered rule?  Additionally, could we send an ICMP message with some further info to inform users for debugging, such as **icmp-admin-prohibited**?